src/components/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Display from './Display';
+import Button from './Button';
+import ButtonPanel from './ButtonPanel';
+
+
+function App() {
+  return (
+    <div>
+
+      <Display value="OOO" />
+      <Button name="boton" color="white" wide />
```

Comment:
Let's make sure to keep the project going without no confusion 😄 , you don't need to add `Button` component in here cause it won't be in here in the later milestone.
kindly remove 👍 
![image](https://user-images.githubusercontent.com/31897434/79173407-5d0cd800-7def-11ea-9875-6e617763a062.png)